Rewriting of the end of install procedures of Dashing/Eloquent/Foxy/Rolling (#845)
ros2/ros2_documentation: source/Tutorials/Configuring-ROS2-Environment.rst

To undo this (to change to another distro) in Linux and macOS, locate your system’s shell startup script and remove the appended source command.

3 Add ``colcon_cd`` to your shell startup script
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> The command ``colcon_cd`` allows to quickly change the current working directory of your shell by reaching the codebase of a desired package.

Reviewer: ```suggestion
The command ``colcon_cd`` allows you to quickly change the current working directory of your shell to the directory of a package.
```

As an example ``colcon_cd some_ros_package`` would quickly bring you to the directory ``~/ros2_install/src/some_ros_package``.

.. tabs::

   .. group-tab:: Linux

      .. code-block:: console

        echo "source /usr/share/colcon_cd/function/colcon_cd.sh" >> ~/.bashrc
        echo "export _colcon_cd_root=~/ros2_install" >> ~/.bashrc

   .. group-tab:: macOS

      .. code-block:: console

        TODO

   .. group-tab:: Windows

      Not yet available

> According to the way you installed ``colcon_cd`` and where your workspace is, the instructions hereabove may vary, please refer to `the documentation <https://colcon.readthedocs.io/en/released/user/installation.html#quick-directory-changes>`__ for more details.

Reviewer: ```suggestion
Depending to the way you installed ``colcon_cd`` and where your workspace is, the instructions above may vary, please refer to `the documentation <https://colcon.readthedocs.io/en/released/user/installation.html#quick-directory-changes>`__ for more details.
```

3 Add ``roscd`` to your shell startup script
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

The command ``colcon_cd`` (alias ``roscd``) allows to quickly change the current working directory of your shell by reaching the codebase of a desired package.
As an example ``roscd some_ros_package`` would quickly bring you to the directory ``~/ros2_install/src/some_ros_package``.

.. tabs::

   .. group-tab:: Linux

      .. code-block:: console

        echo "source /usr/share/colcon_cd/function/colcon_cd.sh" >> ~/.bashrc
        echo "alias roscd='colcon_cd'" >> ~/.bashrc
        echo "export _colcon_cd_root=~/ros2_install" >> ~/.bashrc

   .. group-tab:: macOS

      .. code-block:: console

        TODO

   .. group-tab:: Windows

      Not yet available

According to the way you installed ``colcon_cd`` and where your workspace is, the instructions hereabove may vary, please refer to `the documentation <https://colcon.readthedocs.io/en/released/user/installation.html#quick-directory-changes>`__ for more details.
To undo this in Linux and macOS, locate your system’s shell startup script and remove the appended source command.

> 4 Check environment variables

Reviewer: Two things here:

1.  I don't think we should suggest aliasing this to `roscd`.  Especially when using the bridge, this will collide with the ROS 1 version of the same thing.  I think it is sufficient to point out that `colcon_cd` exists and how to use it.
1.  This seems reasonable to me to add to a beginner tutorial.  But pinging @maryaB-osr to get additional thoughts.

Author: I dropped the alias in https://github.com/ros2/ros2_documentation/pull/845/commits/af35b47a6717b54848bae8739085983caa1158b1. Yes some additional tutorial about how to use it could be nice, I think that part could stay here since it's relevant with the environment config.
